chef/crontab/recipes/default.rb

```diff
@@ -113,7 +113,8 @@
   openaddr-run-ec2-command \
   --role dotmap \
   --hours 16 \
-  --instance-type r3.xlarge \
+  --instance-type r3.large \
```

Comment:
I did! Previously, the larger instance was needed in order to use more temp space, but now temp space is being provided independently of instance storage size.